README.md

```diff
@@ -103,10 +114,16 @@ npm run serve
 ```
 
 ## Configuration
-There are two main ways of configuring the application: Frontend and backend.
-While the defaults were chosen in such a way that they work in a development environment
-(the docker container has different [ones](./docker/settings.json)) they might need change in a production setup.
+### How to configure
+The application is configured at two places:
+1) Environment variables for the backend
+2) A `settings.json` file for the frontend
+
+Defaults defined in the source code were chosen to work in a development environment. The
```

Comment:
This sentence is a bit unclear. What do you meant by "defaults in source code"?

Reply:
Defaults like where backend API and SocketIO server are reachable.